docs/quickstart.md

```diff
@@ -243,6 +236,72 @@ This will generate an ISO on the current directory with the name `elemental-<tim
 </TabItem>
 </Tabs>
 
+
+Note: Much but not all of the configuration provided in a `MachineRegistration` will be read at installation time instead of baked into the installation media. This means that many changes will not require using a new ISO.
```

Comment:
This is still somewhat fuzzy

The only purpose of the embedded-in-the-ISO configuration is to establish the network connection to the management cluster. Everything else will be loaded from this cluster.

Reply:
Ok, I'll drop this line. I added it to suggest that you wouldn't need to rebuild (or redownload) the iso after every change.